### PR TITLE
Internal: Add an Option to the Stretch module to use css variables [ED-11630]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/js/frontend/tools/stretch-element.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = elementorModules.ViewModule.extend( {
 		};
 	},
 
-	stretch() {
+	stretch( useCssVariables = false ) {
```
Comment (louiswol94) [inline assets/dev/js/frontend/tools/stretch-element.js:19]: Add this config to `getDefaultSettings()` in this file.

File: assets/dev/js/frontend/handlers/stretched-element.js [hunk 2]
```diff
@@ -40,6 +40,10 @@ export default class StretchedElement extends Base {
 		return elementorFrontend.isEditMode() || settings.$element.hasClass( this.getStretchedClass() );
 	}
 
+	usesCssVariables() {
+		return false;
+	}
```
Comment (louiswol94) [inline assets/dev/js/frontend/handlers/stretched-element.js:45]: Remove

File: assets/dev/js/frontend/handlers/stretched-element.js [hunk 3]
```diff
@@ -77,7 +81,7 @@ export default class StretchedElement extends Base {
 			return;
 		}
 
-		this.stretchElement.stretch();
+		this.stretchElement.stretch( this.usesCssVariables() );
```
Comment (louiswol94) [inline assets/dev/js/frontend/handlers/stretched-element.js:84]: Remove

**Review by louiswol94 — CHANGES_REQUESTED** [review]

File: assets/dev/js/frontend/tools/stretch-element.js (hunk 1 above)
Comment (hein-obox) [reply assets/dev/js/frontend/tools/stretch-element.js]: Updated.

File: assets/dev/js/frontend/handlers/stretched-element.js (hunk 2 above)
Comment (hein-obox) [reply assets/dev/js/frontend/handlers/stretched-element.js]: Updated.

File: assets/dev/js/frontend/handlers/stretched-element.js (hunk 3 above)
Comment (hein-obox) [reply assets/dev/js/frontend/handlers/stretched-element.js]: Updated.

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**